elements/lisk-transactions/test/register_multisignature_account.spec.ts

```diff
@@ -14,11 +14,11 @@
  */
 import * as cryptography from '@liskhq/lisk-cryptography';
 import { registerMultisignature } from '../src/register_multisignature_account';
-import { MultiSignatureAsset } from '../src/12_multisignature_transaction';
+// import { MultiSignatureAsset } from '../src/12_multisignature_transaction';
 import { TransactionJSON } from '../src/transaction_types';
 import * as time from '../src/utils/time';
 
-describe('#registerMultisignature transaction', () => {
+describe.skip('#registerMultisignature transaction', () => {
```

Comment:
Is it intentional?

Reply:
Yes. https://github.com/LiskHQ/lisk-sdk/issues/4889 will un-skip this.